eduaid_web/src/pages/Text_Input.jsx

```
@@ -1,260 +1,390 @@
-      localStorage.setItem("qaPairs", JSON.stringify(responseData));
+      if (Array.isArray(responseData.output)) {
+        questionsArray = responseData.output.map((q) => normalizeQuestion(q, questionType === "get_boolq" ? "Boolean" : questionType === "get_mcq" ? "MCQ" : "Short"));
+      } else if (responseData.output && responseData.output.questions) {
+        questionsArray = responseData.output.questions.map((q) => normalizeQuestion(q, "MCQ"));
+      } else if (responseData.output_mcq || responseData.output_boolq || responseData.output_shortq) {
+        // "All types" mode — combine them
+        if (responseData.output_mcq?.questions) {
+          questionsArray.push(...responseData.output_mcq.questions.map((q) => normalizeQuestion(q, "MCQ")));
+        }
+        if (responseData.output_boolq?.Boolean_Questions) {
+          responseData.output_boolq.Boolean_Questions.forEach((q) =>
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)